v1/charm-dev.yaml

```diff
@@ -41,16 +41,20 @@ instances:
 
         snap:
           commands:
-          - snap install --classic juju
-          - snap install --classic microk8s
+          # Juju 3.1 is supported until 25.04
+          - snap install juju --channel=3.1/stable
+          - snap install microk8s --channel=1.27-strict/stable
           - snap alias microk8s.kubectl kubectl
           - snap alias microk8s.kubectl k
           - snap install --classic charmcraft
+          - snap install jhack --channel=latest/stable
           - snap install yq
           - snap refresh
 
         runcmd:
-        - DEBIAN_FRONTEND=noninteractive apt -y upgrade
+        - DEBIAN_FRONTEND=noninteractive apt-get remove -y landscape-client landscape-common adwaita-icon-theme humanity-icon-theme
```

Comment:
Duplicate to line 86?

Reply:
BTW, I would also to stay consistent with `apt` (you are adding `apt-get`).

Reply:
Nice spotting! Copy-paste artifact. Fixed in next commit.